test/unit/src/server/handlers/t_login.js

```diff
@@ -23,31 +29,19 @@ describe('processVerifiedAssertion', () => {
     processedError = null;
   });
 
-  it('passes through verification errors', () => {
-    const error = new Error('test');
-    processVerifiedAssertion({}, res, next, error);
-    expect(processedError).toBe(error);
-  });
-
-  it('produces E_SSO_FAIL error if not authenticated', () => {
-    processVerifiedAssertion({}, res, next, null, { 'authenticated': false });
-    expect(processedError.message).toBe(constants.E_SSO_FAIL);
-  });
-
-  it('produces E_NO_SESSION error if the request has no session', () => {
-    processVerifiedAssertion({}, res, next, null, { 'authenticated': true });
-    expect(processedError.message).toBe(constants.E_NO_SESSION);
-  });
-
-  context('with OPENID_TEAMS', () => {
-    before(() => {
-      const overrides = conf.stores['test-overrides'];
-      overrides.set('OPENID_TEAMS', ['team-name']);
-    });
-
-    after(() => {
-      const overrides = conf.stores['test-overrides'];
-      overrides.clear('OPENID_TEAMS');
+  context('when OPENID_TEAMS are configured', () => {
+    let processVerifiedAssertion;
+
+    beforeEach(() => {
+      processVerifiedAssertion = proxyquire(
+        '../../../../../src/server/handlers/login',
+        {
```

Comment:
@bartaz I'm not sure about giving the config helper a method just to help with testing. I'd prefer to replace it with a stub using proxyquire if we can.

Having said that, I'm pretty sure I can reduce the boilerplate here. I'll get back to you.